lib/src/event_controller.dart

```diff
@@ -181,6 +201,33 @@ class EventController<T extends Object?> extends ChangeNotifier {
     notifyListeners();
   }
 
```

Comment:
I'm not sure if this direction is acceptable if it isn't can we please discuss how to implement this feature correctly.